README.md

```diff
@@ -339,7 +342,7 @@ internally.
   - `only` {boolean} If truthy, and the test context is configured to run
     `only` tests, then this test will be run. Otherwise, the test is skipped.
     **Default:** `false`.
-  * `signal` {AbortSignal} Allows aborting an in-progress test.
+  - `signal` {AbortSignal} Allows aborting an in-progress test (except on Node.js v14.x).
```

Comment:
```suggestion
  - `signal` {AbortSignal} Allows aborting an in-progress test (needs `--experimental-abortcontroller` on Node.js v14.x).
```

I believe the previous statement wasn't correct?

Reply:
Well now that I think about it, users are still allowed to pass an `AbortSignal`, which they can't pass if they don't have `AbortController` support themselves anyway. The part that doesn't work is `context.signal`, I should put the warning on that section instead.